### PR TITLE
chore(lint): enable nonamedreturns linter

### LINKED ISSUE #708: nonamedreturns

### PR DESCRIPTION
**Pull request by allnil**
enable nonamedreturns linter
fix code which was highlighted by nonamedreturns

- Fixes #708

### CODE REVIEW

**Review by kehiy — APPROVED** [review]
@allnil Thanks.
LGTM.

@b00f Can you have a final review and merge?

**Review by amirvalhalla — APPROVED** [review]

**Comment by amirvalhalla** [comment]
@allnil 
thanks.
right now, we have 2 PIPs (`PIP-8`,`PIP-9`) that we've implemented them/we're implementing them so you're changes will make conflict with existing PRs etc...
maybe merging your PR takes a bit more time but we will merge it as soon as possible.

File: sortition/vrf.go [hunk 1]
```diff
@@ -16,18 +16,18 @@ func init() {
 }
 
 // Evaluate returns a random number between 0 and max with the proof.
```
Comment (b00f) [inline sortition/vrf.go:18]:
```suggestion
// Evaluate returns a provable random number between [0, max) along with a proof.
// It returns the random number and the proof that can regenerate the random number using
// the public key of the signer, without revealing the private key.
```

File: sortition/vrf.go [hunk 2]
```diff
@@ -16,18 +16,18 @@ func init() {
...
 
-	proof, _ = ProofFromBytes(sig.Bytes())
-	index = GetIndex(proof, max)
+	proof, _ := ProofFromBytes(sig.Bytes())
+	index := GetIndex(proof, max)
 
 	return index, proof
 }
 
 // Verify ensures the proof is valid.
```
Comment (b00f) [inline sortition/vrf.go:29]:
```suggestion
// Verify checks if the provided proof, based on the seed and public key, is valid.
// If the proof is valid, it calculates the random number that
// can be generated based on the given proof.
```

**Review by b00f — CHANGES_REQUESTED** [review]

**Review by b00f — APPROVED** [review]

**Review by b00f — APPROVED** [review]

**Comment by b00f** [comment]
@allnil Thank you. We hope to see you more on Pactus. 
Join us on [Discord](https://discord.gg/H5vZkNnXCu), where we discuss core development further.

**Pull request merged**